documentation/adr/0023-error-format.md

```diff
@@ -0,0 +1,74 @@
+[<-previous](./0022-use-keyword-args.md) |
+next->
+
+# 23 - Error Identifier Format
+
+Date: 2021-Mar-24
+
+## Status
+
+Accepted
+
+## Context
```

Comment:
Alex observed during the discussion that MATLAB has changed its standard errors in the past; that may be worth noting here.

Reply:
How do the standard errors you list below relate to the (current) MATLAB standards?